Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
```

Comment:
Add some context to this commit as to why this is necessary - it's helpful to get an overall summary when looking through the changelog / reviewing.

Reply:
Added this:
"Use newer proxy image that has support for deriving policy name from
the endpoint IP, and for deriving the proxylib parser name from the
network policy.

These changes allow listener configuration to be independent of an
endpoint or an endpoint's network policy."